proto/one/operations/sample/csv_configuration_file.proto

```diff
@@ -4,7 +4,7 @@ import "analyte_results_import_telemetry.proto";
 
 option csharp_namespace ="ONE.Models.CSharp";
 
-message AnalyteResultsImportFile {
+message CsvConfigurationFile {
```

Comment:
Should this name still mention AnalyteResults somewhere?  CsvConfigurationFile might be too generic.

Reply:
I'm open to changing if we have a better alternative. Thoughts @JonathanRobertson-AQI ?